src/allmydata/scripts/runner.py

```diff
@@ -180,19 +222,44 @@ def run():
         from allmydata.windows.fixups import initialize
         initialize()
     # doesn't return: calls sys.exit(rc)
-    task.react(_run_with_reactor)
-
-
-def _setup_coverage(reactor):
+    task.react(
+        lambda reactor: _run_with_reactor(
+            reactor,
+            configFactory(),
+            argv,
+            stdout,
+            stderr,
+        ),
+    )
+
+
+def _setup_coverage(reactor, argv):
```

Comment:
I wonder if "coverage" has a better way yet? (I feel like I've explored "why isn't sub-process coverage working" too many times already though :/ )

Reply:
Last time I looked at it, it didn't, and the ticket and PR I filed against it have seen zero response...